src/configuration/js-team/linux64.js

```diff
@@ -0,0 +1,319 @@
+import { SETTINGS } from '../appDefaults';
+
+const { TALOS_FRAMEWORK_ID, RAPTOR_FRAMEWORK_ID, JSBENCH_FRAMEWORK_ID } = SETTINGS.frameworks;
```

Comment:
```suggestion
const { frameworkIds } = SETTINGS;
```
Use this approach instead.